src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -331,3 +331,66 @@ def load_arguments(self, _):
     with self.argument_context('monitor activity-log alert scope remove') as c:
         c.argument('scopes', options_list=['--scope', '-s'], nargs='+')
     # endregion
+
+    # region LogAlerts
+    with self.argument_context('monitor log alert') as c:
+        c.argument('rule_name', options_list=['--name', '-n'], id_part='name')
+
+    with self.argument_context('monitor log alert create') as c:
+        c.argument('location')
+        c.argument('disable', action='store_true')
```

Comment:
Please use get_three_state_flag() as the `arg_type` for such argument.

Reply:
Getting error on adding this parameter for "disable":
File "C:\Program Files (x86)\lib\argparse.py", line 1353, in add_argument
    action = action_class(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'choices'

Reply:
As discussed offline `action='store_true'` is the correct kwarg to be used here.